code/index.html

```diff
@@ -1,22 +1,84 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Knitty&#9829News</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale: 1.0">
+    <!--Google fonts-->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Yusei+Magic&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <header>
+    <header class="header">
       <!-- Some kind of logo and navbar -->
+      <div class="flex-container">
+        
+        <div class="logo">    
+          <img src="./KN_logo.svg" alt="logotype KnittyNews" />
+        </div>
+
+        <div class="nav">
+          <div class="nav-item"><a href="#">About</a></div>
+          <div class="nav-item"><a href="#">Archieve</a></div>
+          <div class="nav-item"><a href="#">Forum</a></div>
+          <div class="nav-item"><a href="#">Local</a></div>
+          <div class="nav-item"><a href="#">Contact</a></div>
+       </div>
+
+      </div>
     </header>
 
+
+  <div class="main">
+    
     <section class="big-news">
       <!-- The big news content -->
+        <div class="big-news-item1">
+          <a href="#"><img id="img-big" src="https://images.unsplash.com/photo-1499728603263-13726abce5fd?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80" alt="image with phrase mindfulness" /></a>
+        </div>
+    
+        <div class="big-news-item2">
+          <a href="#"><h1 id="big-news-h1">Healthboost? <br>-positive effects of knitting</h1></a>
+          <p id="big-news-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. A vel, similique inventore necessitatibus eaque dolores. Molestiae quae aliquam voluptates totam neque alias asperiores, numquam ab, quod tempore, voluptatum labore voluptas.</p>
```

Comment:
Ids are mostly used for JavaScript, so to answer your question you could stick to style classes :) Read more in this weeks general feedback document :👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)